sapi/phpdbg/phpdbg_prompt.c

```diff
@@ -1294,7 +1294,7 @@ PHPDBG_API const char *phpdbg_load_module_or_extension(char **path, char **name)
 			goto quit;
 		}
 
-		*name = new_extension->name;
+		*name = (char *) new_extension->name;
```

Comment:
Wouldn't it be better to make the param `const char **name`, or is that not possible?

Reply:
> Wouldn't it be better to make the param `const char **name`, or is that not possible?

You are right.